packages/11ty/content/_assets/javascript/lit/lightbox.js

```diff
@@ -0,0 +1,342 @@
+import { LitElement, css, html } from 'lit';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+
+class Lightbox extends LitElement {
+  static properties = {
+    currentId: { attribute: 'current-id', type: String },
+    figures: {
+      converter: ((value) => value ? JSON.parse(decodeURIComponent(value)) : null)
+    },
+    height: { type: Number },
+    imageDir: { attribute: 'image-dir', type: String },
+    isInsideOpenModal: { attribute: 'is-inside-open-modal', type: Boolean },
+    width: { type: Number }
+  }
+
+  static styles = css`
+    .q-lightbox {
+      --atlas-z-index: 0;
+      font-family: "IBM Plex Sans Condensed",sans-serif;
+      color: white;
+    }
+
+    .q-lightbox,
+    .q-lightbox__slide,
+    .q-lightbox__image {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      top: 0;
+      left: 0;
+    }
+
+    .q-lightbox__slide {
+      transform: translateY(-100%);
+      opacity: 0;
+      transition: transform 0s 0.4s, opacity 0.4s linear;
+    }
+
+    .current.q-lightbox__slide {
+      transform: translateY(0);
+      opacity: 1;
+      transition: transform 0s, opacity 0.4s linear;
+    }
+
+    .q-lightbox__image img {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      object-fit: contain;
+    }
+
+    .q-lightbox__fullscreen-button {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 30px;
+      height: 30px;
+      margin-top: 10px;
+      margin-left: 10px;
+      padding: 0;
+      border: 0;
+      cursor: pointer;
+      background-color: rgba(0, 0, 0, 0.5);
+      background-repeat: no-repeat;
+      background-position-x: 2px;
+      background-position-y: 2px;
+      background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABoAAAA0AgMAAADpgsAbAAABS2lUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPD94cGFja2V0IGJlZ2luPSLvu78iIGlkPSJXNU0wTXBDZWhpSHpyZVN6TlRjemtjOWQiPz4KPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iQWRvYmUgWE1QIENvcmUgNS42LWMxNDAgNzkuMTYwNDUxLCAyMDE3LzA1LzA2LTAxOjA4OjIxICAgICAgICAiPgogPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4KICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIi8+CiA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgo8P3hwYWNrZXQgZW5kPSJyIj8+LUNEtwAAAARnQU1BAACxjwv8YQUAAAABc1JHQgCuzhzpAAAACVBMVEUAAAD///////9zeKVjAAAAAnRSTlMAgJsrThgAAAAySURBVBjTY2AgFmitYFrVgERrMDAw4aMJAUL60e2jNiDaP2wrIf7hWgXRiIOGqaOufwA6TR6/lErWHQAAAABJRU5ErkJggg==);
```

Comment:
base64-encoded icons stolen from `magnific-popup` until we can add our own bundled svg/png asssets